airbyte-webapp/.eslintrc

```diff
@@ -33,5 +31,13 @@
       "warn"
     ]
   },
-  "parser": "@typescript-eslint/parser"
+  "parser": "@typescript-eslint/parser",
+  "overrides": [
```

Comment:
:information_source: Disable requiring `import` syntax in scripts, since they are running on node directly so should use `require` for now.